app/src/main/java/com/google/android/stardroid/activities/dialogs/EulaDialogFragment.java

```diff
@@ -56,7 +56,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
     TextView apologyTextView = (TextView) view.findViewById(R.id.language_apology_box_text);
     apologyTextView.setText(formattedApologyText, TextView.BufferType.SPANNABLE);
 
-    String eulaText = String.format(parentActivity.getString(R.string.eula_text), getVersionName());
+    String eulaText = String.format(parentActivity.getString(R.string.eula_text) +
```

Comment:
This isn't quite right. The version number is supposed to be interpolated into the text.
That said....due to refactorings of the text it looks like there's no longer a placeholder for it in the EULA text.  That's not the end of the world, but we could just remove the String.format and getVersionName altogether.

Reply:
@jaydeetay So you want `eula_text` to be concatenated with the version name ? And get assigned to `eulaText` right?

Reply:
Not really. The original text would have had a placeholder in it. Something like "EULA for version: %s.  Blah blah blah...".  So concatenating isn't the right thing to do. Over time it looks like the version number has been removed from the text (or perhaps it wasn't present) so the String.format isn't doing anything.  I think it's harmless, but it's also redundant.